multiuser/machine-auth/che-multiuser-machine-authentication/src/test/java/org/eclipse/che/multiuser/machine/authentication/server/MachineLoginFilterTest.java

```diff
@@ -118,7 +121,7 @@ private void setUp() throws Exception {
   public void testProcessRequestWithValidToken() throws Exception {
     machineLoginFilter.doFilter(getRequestMock(), responseMock, chainMock);
 
-    verify(keyManagerMock).getOrCreateKeyPair(eq(WORKSPACE_ID));
+    verify(keyManagerMock, atLeastOnce()).getOrCreateKeyPair(eq(WORKSPACE_ID));
```

Comment:
what is the reason for adding atLeastOnce at this test class?

Reply:
signing key may be requested more than one time - on getUser, on getSubject.